src/pages/home/report/ReportActionItemThread.tsx

```diff
@@ -46,7 +46,7 @@ function ReportActionItemThread({numberOfReplies, icons, mostRecentReply, childR
             <PressableWithSecondaryInteraction
                 onPress={() => {
                     Report.navigateToAndOpenChildReport(childReportID);
-                    Timing.start(CONST.TIMING.SWITCH_REPORT);
+                    Timing.start(CONST.TIMING.SWITCH_REPORT_THREAD);
```

Comment:
And, by the way, why do we start the timer after initiating a navigation? Shouldn't we call `Timing.start` before `navigateToAndOpenChildReport`? And could this be the reason for the high timings in the graph?

Reply:
I think we can discuss but its NAB for this PR

Reply:
@mountiny Do you think we can create another issue for this and discuss? Or if there is another related issue on analytics we can add this issue there and discuss.